packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -613,14 +615,16 @@ describe('<ButtonBase />', () => {
       const button = getByRole('button');
 
       simulatePointerDevice();
-      focusVisible(button);
+      await act(async () => {
+        focusVisible(button);
+      });
```

Comment:
if `focusVisible` is not using act then let's make sure it does and add `await null` so that it  stands out.

We'll be able to remove `await null` before merging anyway.

Reply:
This is fixed to

```js
await focusVisible(button);
```